package.json

```diff
@@ -7,28 +7,30 @@
   "engines": {
     "node": ">=12"
   },
-  "version": "0.0.0-semantically-released",
+  "version": "3.4.0",
   "main": "lib/commonjs",
   "module": "lib/es2015",
   "typings": "lib/index.d.ts",
   "files": [
-    "config",
-    "lib",
+    "config/**/*",
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map",
+    "lib/template/**/*",
```

Comment:
We were previously bundling `tsbuildinfo`s 🤦

This chicanery appears to get us down from ~1.5MB to ~0.5MB